### DISCUSSION TITLE
Portainer hasn't made any 3rd Party Auth-Methods "Business Only"

### DISCUSSION THREAD

**Discussion by EpicLPer** [hassio-addons/addon-portainer#135] [General]
It appears that Portainer already got in touch with you, I'm mainly opening this issue for others to read about this too and be informed about what's going on :)

-----

# Problem/Motivation

> In the latest Changelog it was said that Portainer now moved the previously used Auth method to their "Business" model. This seems to have been a bit of a miscommunication.

## Expected behavior

> X

## Actual behavior

> Portainer only deprecated the `-no-auth` option due to misuse by their users.
More infos can be read here: https://twitter.com/portainerio/status/1433914478534430720

## Steps to reproduce

> X

## Proposed changes

> X

**Comment by EpicLPer** [comment]
I've read through a bunch of other issues now to see what's been going on and saw that apparently "Portainer lies there"? Is this for real?

**Comment by ncresswell** [comment]
I would love to know what features we moved from CE to Business, we have not moved ANY features across at all. Sure, we deprecate features from CE (--no-auth), but we didnt do that to move it to Business (Business doesnt have that functionality). If someone can point to a feature we moved, i would happily investigate, but we do not lie, never have, never will.

**Comment by frenck** [comment]
They did lie and changed models around for profit. While I do understand those things happen, they simply deny it every time. 

They have removed the ability to disable authentication so other methods can't be used/supplied (eg a reverse proxy, hence the add-on now must use double auth for no sane reason at all). The sole sane reason for this change can be found in the fact additional authentication modules are a paid feature.

I would not recommend doing business with Portainer for that reason.
